src/main/java/racing/RacingCar.java

```diff
@@ -0,0 +1,51 @@
+package racing;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingCar {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static InputView inputView = new InputView();
+    private static ResultView resultView = new ResultView();
+    private static int[] position;
+
+    public static void racing() {
+        inputView.input(scanner);
+
+        position = new int[inputView.getCars()];
+
+        resultView.printTitle();
+
+        for(int i=0 ; i< inputView.getTimes() ; i++) {
+            getResult(inputView.getCars());
+            resultView.printBlank();
+        }
+    }
+
+    private static void getResult(int cars) {
+        for(int j = 0; j < cars ; j++) {
+            position[j] = plusCount(position[j], getRandomNum());
+            resultView.printResult(position[j]);
+        }
+    }
+
+    public static int plusCount(int count, int number) {
+        if(isFourOrMore(number)) {
```

Comment:
코드 컨벤션 한번 보고 가시면 좋겠습니다 😄 
```suggestion
        if (isFourOrMore(number)) {
```

그리고 추가적으로 만약 자동차가 전진하는 조건이 `5`이상으로 변경되면 이 `method` 명도 변경되어야 할 것 같네요 😅 
뭔가 좀 더 요구사항을 잘 나타낼 수 있는 이름이면 좋겠습니다 🙏